src/components/baseComponent.ts

```diff
@@ -6,6 +6,30 @@
  */
 
 import { LitElement, PropertyValues } from 'lit-element';
+
+/**
+ * Defines media query based on component width
+ *
+ * @export
+ * @enum {string}
+ */
+export enum ComponentMediaQuery {
```

Comment:
Should we rename these to something like: narrow, wide, etc?

Sizes are not exclusive to form factor

Reply:
I prefer to use the type of the device 'cause we could have say "dual screen" in the future. The size might be the same but the design would be different.